web_listview_range_select/static/src/js/web_listview_range_select.esm.js

```diff
@@ -0,0 +1,105 @@
+/** @odoo-module */
```

Comment:
Instead of creating this new `web_listview_range_select.esm.js` file, can't you just reuse the existing `web_listview_range_select.js` file? This way the commit history will be preserved.

Reply:
If we use same file name, we are getting pre-commit error while pushing code.

Reply:
Yes need to be .esm.js to avoid pre-commit errors. I resolve this conversation